cmd/podman/oci-trace-hook/trace.go

```diff
@@ -0,0 +1,238 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/binary"
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	bpf "github.com/iovisor/gobpf/bcc"
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+	seccomp "github.com/seccomp/libseccomp-golang"
+)
+
+type event struct {
```

Comment:
Can we get a comment here to explain what this is/what it's used for?

Reply:
Still valid but I am closing as there's another conversation addressing the same issue.